### ISSUE TITLE
Cannot invoke value of type 'EasyDialog.Builder.Type' with argument list '(ViewController)'

### ISSUE THREAD

**Issue by 0xkalinowski** [bontoJR/EasyDialogs#1]
Hey,

I'm tryin' to show alert view:

```
            EasyDialog.Builder(self)
                .title(title: "Hello World")
                .text(content: "This is a basic dialog")
                .space(ofSize: 4)
                .addButton(title: "Ok") { dialog in
                    dialog.dismiss(animated: true)
                }
                .build()
                .show()
```
but getting this error:

`Cannot invoke value of type 'EasyDialog.Builder.Type' with argument list '(ViewController)'`

**Comment by bontoJR** [comment]
Ouch, it's a problem with `public` and `private` in a Framework.

I quickly solved it and pushed a new version `0.9.2`, thanks for reporting it!

**Comment by 0xkalinowski** [comment]
awesome!
